lisa/sut_orchestrator/hyperv/schema.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from dataclasses import dataclass, field
+from typing import List
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+
+
+@dataclass_json
+@dataclass
+class HypervServer:
+    address: str
+    username: str
+    password: str
+
+
+@dataclass_json
+@dataclass
+class ExtraArgs:
+    command: str
+    args: str
+
+
+@dataclass_json()
+@dataclass
+class VHDSourceSchema(schema.TypedSchema, schema.ExtendableSchemaMixin):
```

Comment:
Name it `BuildSchema`, so it's more close to BareMetal. It doesn't need have vhd in the name.

Reply:
I guess this comment can be resolved since we discussed this topic in the other comment.